crates/gettext-maps/build.rs

```diff
@@ -0,0 +1,138 @@
+use std::{
+    env,
+    ffi::OsString,
+    path::{Path, PathBuf},
+    process::Command,
+};
+
+fn main() {
+    let cargo_target_dir = fish_build_helper::get_target_dir();
+    // FISH_BUILD_DIR is set by CMake, if we are using it.
+    let fish_build_dir =
+        option_env!("FISH_BUILD_DIR").unwrap_or(cargo_target_dir.to_str().unwrap());
```

Comment:
since this variable is also used in our workspace build.rs,
let's maybe extract a function to fish-build-helper.
It should return a non-canonicalized path.
I guess I'd make it return `Cow<'static, Path>`.